client/components/ZoopPage.tsx

```diff
@@ -1,13 +1,41 @@
 import React from "react";
+import { useGetZoopQuery } from "../features/api";
+import { useParams } from "react-router-dom";
+import ZoopDetails from "./ZoopDetails";
+import { skipToken } from "@reduxjs/toolkit/query";
```

Comment:
What does this do?

Reply:
Per the docs: 
-RTK Query provides the ability to conditionally skip queries from automatically running using the skip parameter as part of query hook options
-RTK Query provides a skipToken export which can be used as an alternative to the skip option in order to skip queries, while remaining type-safe. When skipToken is passed as the query argument to useQuery, useQueryState or useQuerySubscription, it provides the same effect as setting skip: true in the query options, while also being a valid argument in scenarios where the arg might be undefined otherwise.

https://redux-toolkit.js.org/rtk-query/usage-with-typescript#skipping-queries-with-typescript-using-skiptoken

I was getting this error when passing the id to useGetZoopQuery: 
```Argument of type 'string | undefined' is not assignable to parameter of type 'string | unique symbol'. Type 'undefined' is not assignable to type 'string | unique symbol'.ts(2345)```